### DISCUSSION TITLE
Why is the accuracy I achieved using the KiTS 23 dataset not as good as that reported in the paper?

### DISCUSSION THREAD

**Discussion by abcsddffffsxx** [anthonyweidai/SvANet#12] [Q&A]
Prepare your data: Arrange the images and masks in the same folder, structured as follows:：
${SvANet_ROOT}
|-- dataset
`-- |-- KiTS23
    `-- |--- train
            |--- case_00000_132.jpg
            |--- case_00000_203.jpg
            |--- ...
            |--- case_00272_247.jpg
            |--- ...
        |--- val
            |--- case_00009_28.jpg
            |--- case_00009_37.jpg
            |--- ...
            |--- case_00514_441.jpg
            |--- ...
        |--- mask
            |--- case_00000_132.png
            |--- case_00000_203.png
            |--- ...
            |--- case_00514_441.png
            |--- ...
            
           Is there code available to generate this structure? I'm using my own method to generate data, which is slightly different from this. Could this be the reason why my accuracy is not as good as mentioned in the paper?
   
![论文结果](https://github.com/user-attachments/assets/175d88f8-26d0-4ca5-b848-ec2fd7a39151)
The results I obtained from running the experiment based on the paper still show a significant gap compared to the results reported in the paper.

**Comment by GOfBlue** [comment]
Hello, when I run niftiFileExtractor.py to process KiTS23, ColourCode = COLOUR_CODES['kits23'] error occurs. Could you share the colourPlatte.py you wrote? Thank you very much!

**Comment by abcsddffffsxx** [comment]
> 您好，当我运行 niftiFileExtractor.py 来处理 KiTS23 时，出现 ColourCode = COLOUR_CODES['kits23'] 错误。您能分享一下您写的 colourPlatte.py 吗？谢谢！

COLOUR_CODES = {
    'twoclasses': [
        [0, 0, 0], # background
        [255, 255, 255]
        ],
    **dict.fromkeys(
        [
            'default', 
            'semspermna', 'semspermnav1', 'semspermnav2', 
            'atlas', 'fives',
            'kits23'  # 添加 'kits23'
        ], 
        list(map(list, imgviz.label_colormap(256)))
    ),
    }
